system/jlib/jfile.cpp

```diff
@@ -7643,3 +7643,110 @@ IAPICopyClient * createApiCopyClient(IStorageApiInfo * source, IStorageApiInfo *
     }
     return nullptr;
 }
+
```

Comment:
I did not review this bit in detail as it looks like it's copied from my prior PR.

Reply:
yep, your original commit is the 1st of the 2 commits in this PR, I made minor changes only to CBlockedFileIO